ccswbs/gus
# Use fragments in GraphQL queries and refactor Page and Widget

# Summary of changes
- Use graphql fragments to shorten code
- Create a new Page template with refactored code
- Create a new Widget component with refactored code

## Frontend
- Shorter code that's easier to read

## Backend
- None

# Test Plan

- Run gatsby using https://api.liveugconthub.uoguelph.dev/ as DRUPAL_BASEURL
- Confirm that all widgets still work as expected
- Confirm that page displays as expected
